src/main/webapp/about.jsp

```diff
@@ -32,38 +33,35 @@
     <a href="/about.jsp">About</a>
   </nav>
 
-  <div id="container">
-    <div
-      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+  <div id = "container">
+    <div style = "width:90%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
-      <h1>About the CodeU Chat App</h1>
-      <p>
-        This is an example chat application designed to be a starting point
-        for your CodeU project team work. Here's some stuff to think about:
-      </p>
+       <h1>What is the CodeU Chat App?</h1>
+       <p>
+         This java-based app is aimed at simple and quick communication between teams.
+         Built by CodeU'ers, this app is a semester project that allows us to practice
+         our software engineering and team works skills. Here, you can find information on
+         recently added features and improvements.
+       </p>
 
-      <ul>
-        <li><strong>Algorithms and data structures:</strong> We've made the app
-            and the code as simple as possible. You will have to extend the
-            existing data structures to support your enhancements to the app,
-            and also make changes for performance and scalability as your app
-            increases in complexity.</li>
-        <li><strong>Look and feel:</strong> The focus of CodeU is on the Java
-          side of things, but if you're particularly interested you might use
-          HTML, CSS, and JavaScript to make the chat app prettier.</li>
-        <li><strong>Customization:</strong> Think about a group you care about.
-          What needs do they have? How could you help? Think about technical
-          requirements, privacy concerns, and accessibility and
-          internationalization.</li>
-      </ul>
+       <div class = "category">
+         <h2>Recently Added Features</h2>
```

Comment:
Nice touch with this. It would be cool to keep this up to date as we add more features.

Reply:
I definitely will!